datalake/src/main/java/com/sequenceiq/datalake/cm/ClouderaManagerProxiedClientFactory.java

```diff
@@ -0,0 +1,32 @@
+package com.sequenceiq.datalake.cm;
+
+import com.cloudera.api.swagger.client.ApiClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.concurrent.TimeUnit;
+
+@Component
+public class ClouderaManagerProxiedClientFactory {
```

Comment:
altogether it would be nice if you could use https://github.com/hortonworks/cloudbreak/blob/master/client-cm/src/main/java/com/sequenceiq/cloudbreak/cm/client/ClouderaManagerApiClientProvider.java instead of this class

Reply:
As mentioned in previous comment, can we look into this as a separate ticket.